setup.cfg

```diff
@@ -8,8 +8,7 @@ env =
 	D:prometheus_multiproc_dir=/tmp/prometheus_client/
 
 [flake8]
-ignore = E501
-max-line-length = 160
+max-line-length = 120
```

Comment:
100

Reply:
In that case all files need to be refactored, why do we need exact 100?

Reply:
We can use 79 if you want. 100 is a compromise because it's a standard for JS files (which we have enough in the project), so we can use the same max length for all the files. 
Why do we need 120?